endorsement/dao/shared_drive.py

```diff
@@ -70,48 +104,311 @@ def get_drive_quota(a):
     """
     return a shared drive quota model from OrgUnit
     """
-    drive_quota, _ = SharedDriveQuota.objects.get_or_create(org_unit=a.OrgUnit)
+    defaults = {"org_unit_name": a.org_unit_name}
+    try:
+        defaults["quota_limit"] = a.quota_limit
+    except ValueError:
+        "Drive has no quota set. Probably in invalid org unit."
```

Comment:
The expectation is this will get "fixed" during reconciliation, but it's a little awkward prior to our cutover when the majority of the share drives are in an OU with no quota

Reply:
I'd have to confer with Ken on that to see if there's a quota currently being enforced.

I'm aware of 3 OUs

- the default UW one, where most drives are
- one that follows a different naming scheme but appears to be compatible with a new "XXXGB" OU
- the one for drives pending deletion